GAE/src/com/gallatinsystems/user/app/gwt/client/PermissionConstants.java

```diff
@@ -37,6 +37,6 @@
     public static final String EDIT_TOKENS = "EDIT TOKENS";
     public static final String VIEW_MESSAGES = "VIEW MESSAGES";
     public static final String EDIT_IMMUTABLITY = "EDIT IMMUTABILITY";
-    public static final String ADMIN = "ADMIN";
-    public static final String SUPER_ADMIN = "SUPER_ADMIN";
+    public static final String ROLE_ADMIN = "ROLE_ADMIN";
```

Comment:
Why one is called ROLE_ADMIN? and the other SUPER_ADMIN, should it be called ROLE_SUPER_ADMIN?

Reply:
No, strangely super-admin is not a role.